app/controllers/api/storage.php

```diff
@@ -245,6 +246,155 @@
         $response->noContent();
     });
 
+App::post('/v1/storage/buckets/:bucketId/files')
+    ->desc('Create File')
+    ->groups(['api', 'storage'])
+    ->label('scope', 'files.write')
+    ->label('event', 'storage.files.create')
+    ->label('sdk.auth', [APP_AUTH_TYPE_SESSION, APP_AUTH_TYPE_KEY, APP_AUTH_TYPE_JWT])
+    ->label('sdk.namespace', 'storage')
+    ->label('sdk.method', 'createFile')
```

Comment:
How do we ensure backwards compatibility?

Reply:
My idea is to keep old endpoints for X time, use migration to move all the old file to a bucket called `X`, and make sure the old endpoint treat any request if it was for bucket `X`. What do you say?

This might require us to move file locations during migration, so the delete worker could delete 1 parent dir instead of each file.